manifest.json

```diff
@@ -14,7 +14,9 @@
     "vtex.styleguide": "9.x",
     "vtex.css-handles": "0.x",
     "vtex.checkout-container": "0.x",
-    "vtex.checkout-payment": "0.x"
+    "vtex.checkout-payment": "0.x",
+    "vtex.checkout-profile": "0.x",
+    "vtex.order-manager": "0.x"
```

Comment:
⛏ Sorry about nitpicking about this, but could you sort the dependencies alphabetically? 😬

Reply:
sure thing!